baseplate/experiments/__init__.py

```diff
@@ -134,25 +138,43 @@ def variant(self, name, user=None, bucketing_event_override=None,
         :rtype: :py:class:`str`
         :return: Variant name if a variant is active, None otherwise.
         """
+
         experiment = self._get_experiment(name)
 
         if experiment is None:
             return None
 
         inputs = dict(kwargs)
+
+        """ clean up user id. If not a baseplate user, we risk inconsistency."""
```

Comment:
style nit: if it's a comment just write it as a comment rather than a string literal